eZ Platform/Admin UI/README.md

```diff
@@ -1,7 +1,76 @@
-# About this folder
+# v2: Symfony 3 Hybrid Prototype: Admin UI Requirements
 
-This is the Requirements & Specifications repository for eZ Platform.
+## Prototype Objective
+"As an eZ Engineer, I want...
 
-https://github.com/ezsystems/specs
+To create a functioning, experimental combination of Symfony3, eZ Platform, and
```

Comment:
I would maybe make it a bit more obvious in this sentence that the core of the admin will be symfony, with server side routes, etc. But that it will be part of an hybrid architecture that will facilitate the embedding of rich frontend modules that can communicate with the server and between each other.